index.js

```diff
@@ -12,24 +12,11 @@ const defaults = {
   ignoreUrl: false,
   temporary: false,
   redirectMethods: ['GET', 'HEAD'],
-  internalRedirectMethods: []
+  internalRedirectMethods: [],
+  specCompliantDisallow: false
```

Comment:
Can you please add test case for this option? This should be pretty straight forward if you look at test suit.